test/markup/perl/class.expect.txt

```diff
@@ -0,0 +1,10 @@
+<span class="hljs-keyword">use</span> <span class="hljs-number">v5.38</span>;
+
+<span class="hljs-class"><span class="hljs-keyword">class</span> <span class="hljs-title">Example</span>::<span class="hljs-title">Subclass</span><span class="hljs-attribute"> :isa(Example::Base 2.345)</span> </span>{
+    <span class="hljs-keyword">field</span> $_config<span class="hljs-attribute"> :param</span><span class="hljs-attribute"> :param</span><span class="hljs-attribute"> :param</span>;
+    <span class="hljs-keyword">field</span> $_name<span class="hljs-attribute"> : param</span> = <span class="hljs-string">&#x27;Test&#x27;</span>;
+
+    <span class="hljs-function"><span class="hljs-keyword">method</span> <span class="hljs-title">test</span>() </span>{
+        $_name == <span class="hljs-string">&#x27;Test&#x27;</span> ? <span class="hljs-string">&#x27;y&#x27;</span> : <span class="hljs-string">&#x27;n&#x27;</span>;
```

Comment:
Would we want to tag the identifier portion of these as variables?  Ie, `$_name` ?

Reply:
Oh! I completely missed it. Yes! It should be tagged. I'll get it done.

Reply:
So, it turns out that all tests were ignoring the presence of variables due to the missing `variable` scope and changing it also required to change Mojolicious (`src/languages/mojolicious.js`) test case, which also had a conflict with the `%=` tag. I pushed 2 new commits to get variables to work.